io.catenax.fleet.claim_data/2.0.0/ClaimData.ttl

```diff
@@ -0,0 +1,324 @@
+   samm:preferredName "Country Code Characteristic"@en ;
+   samm:description "ISO 3166-1 alpha-3 – three-letter country codes "@en ;
+   samm:see <https://www.iso.org/iso-3166-country-codes.html> ;
+   samm:dataType xsd:string .
+
+:CountryCodeRegularExpression a samm-c:RegularExpressionConstraint ;
+   samm:preferredName "Country Code Regular Expression"@en ;
+   samm:description "Regular Expression that ensures a three-letter code "@en ;
+   samm:value "^[A-Z][A-Z][A-Z]$" .
+
+:ClaimedPart a samm:Entity ;
+   samm:preferredName "Part"@en ;
+   samm:description "one specific part"@en ;
+   samm:properties ( [ samm:property :amountOfReplacedParts; samm:optional true ] [ samm:property :catenaXClaimPartId; samm:optional true ] [ samm:property :isPartCausal; samm:optional true ] [ samm:property :isPartReplaced; samm:optional true ] [ samm:property :partName; samm:optional true ] [ samm:property :partNumber; samm:optional true ] [ samm:property :partTreatement; samm:optional true ] [ samm:property :serialNumber; samm:optional true ] [ samm:property :spareParts; samm:optional true ] [ samm:property :supplierId; samm:optional true ] ) .
+
+:sessionId a samm:Property ;
+   samm:preferredName "Session identifier"@en ;
+   samm:description "A oem-specific session identifier that is unique in this company."@en ;
+   samm:characteristic :UniqueID ;
+   samm:exampleValue "3747429FGH382923974682_Session_2023-11-04T08:00:45" .
+
+:amountOfReplacedParts a samm:Property ;
+   samm:preferredName "AmountOfReplacedParts"@en ;
+   samm:description "counter for non-serial parts which have been replaced"@en ;
+   samm:characteristic :AmountOfReplacedPartsCharacteristic ;
+   samm:exampleValue "2"^^xsd:nonNegativeInteger .
+
+:catenaXClaimPartId a samm:Property ;
+   samm:preferredName "Catena-X Claim Part ID"@en ;
+   samm:description "Catena-X ID of the part that is involved in the claim."@en ;
+   samm:characteristic ext-uuid:UuidV4Trait ;
+   samm:exampleValue "urn:uuid:580d3adf-1981-44a0-a214-13d6ceed9000" .
+
+:isPartCausal a samm:Property ;
+   samm:preferredName "IsPartCausal"@en ;
+   samm:description "Flag is set to true if part was causing the problem-otherwise false."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:isPartReplaced a samm:Property ;
+   samm:preferredName "IsPartReplaced"@en ;
+   samm:description "Flag is set to true if part was replaced-otherwise false."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:partName a samm:Property ;
+   samm:preferredName "Part name"@en ;
+   samm:description "Part name of this part."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Gear box control unit." .
+
+:partNumber a samm:Property ;
+   samm:preferredName "Part number"@en ;
+   samm:description "Internal part number of the part. A part number is NOT unique."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "12345" .
+
+:partTreatement a samm:Property ;
```

Comment:
typo with `Treatement`. please fix property name and `preferredName`

Reply:
Fixed